e2e-tests/playwright/e2e/plugins/quay/quay.spec.ts

```diff
@@ -1,18 +1,25 @@
-import { test } from '@playwright/test';
+import { Page, test, chromium, firefox, expect } from '@playwright/test';
 import { UIhelper } from '../../../utils/UIhelper';
 import { Common } from '../../../utils/Common';
 import { ImageRegistry } from '../../../utils/quay/quay';
+import { UIhelperPO } from '../../../support/pageObjects/global-obj';
 
-test.describe('Test Quay.io plugin', () => {
+test.describe.serial('Test Quay.io plugin', () => {
+  let page: Page;
+  let uiHelper: UIhelper;
   const QUAY_REPOSITORY = 'janus-idp/backstage-showcase';
 
-  test.beforeEach(async ({ page }) => {
+  test.beforeAll(async ({ browserName }) => {
```

Comment:
`loginAsGuest` runs really fast I think we can keep `beforeEach` and avoid all the browserType stuff